src/organisms/footer/index.jsx

```diff
@@ -3,7 +3,6 @@ import React from "react";
 const Footer = () => {
 	return (
 		<footer>
-			<div>© 2022 by Me</div>
 		</footer>
 	);
 };
```

Comment:
Looks like you don't need this component at this point because it's empty!?
If so, you can remove it.